types/tools/block-tool.d.ts

```diff
@@ -19,9 +19,9 @@ export interface BlockTool extends BaseTool {
   /**
    * Process Tool's element in DOM and return raw data
    * @param {HTMLElement} block - element created by {@link BlockTool#render} function
-   * @return {BlockToolData}
+   * @return {BlockToolData | Promise<BlockToolData>}
    */
-  save(block: HTMLElement): BlockToolData;
+  save(block: HTMLElement): BlockToolData | Promise<BlockToolData>;
```

Comment:
BlockTool.save() supports async/await, so support Promise also in type definition.